Cargo.toml

```diff
@@ -11,50 +11,50 @@ categories = ["command-line-utilities", "network-programming", "web-programming:
 edition = "2021"
 
 [profile.release]
+codegen-units = 1
 lto = true
 opt-level = 'z'
-codegen-units = 1
 panic = 'abort'
 
 [dependencies]
-actix-web = "4"
 actix-files = "0.6"
 actix-multipart = "0.4"
+actix-web = "4"
 actix-web-httpauth = "0.8"
-maud = "0.23"
-yansi = "0.5"
-simplelog = "0.12"
```

Comment:
If you want to, you could make another PR to only order these around so that we keep history clean and total changes low.

Reply:
I ran `cargo sort` on master to minimize the difference here. However, that means you're going to have to rebase once.

Reply:
Done.